### DISCUSSION TITLE
Request: Simple 'Goin Round' Feature

### DISCUSSION THREAD

**Discussion by SteeltonMC** [EssentialsX/Essentials#4373] [Ideas and Feature Suggestions] [answered]
Hello, long story short, there was a plugin I used to have like 8 years ago that was called Goin'Round. Long abandoned and forgotten. I can not find ANYTHING similar or replaceable. 

The idea was simple, automatically TP (yourself) to all online players for a given time.

Example: /Essentials GoinRound 20
This would teleport (you) to each player on the server, for 20 seconds each. 

This lets Mods or Admins visit each player automatically without having to manually TP to each player over and over. Very simple and helpful!

Please let me know what you think.

**Comment by JRoy** [comment, marked as answer]
This is out of the scope of Essentials.
